grommet/grommet
# fix: only show focus on keyboard use

Inspired by [this](https://nelo.is/writing/styling-better-focus-states/) I'm adding support for `:focus-visible` if available so that we only show the focus ring if you are using the keyboard

#### What does this PR do?

Show focus only when using keyboard

#### Where should the reviewer start?
StyledButton

#### What testing has been done on this PR?
manual

#### How should this be manually tested?
Make sure you see the focus outline when using the keyboard and you dont see it when clicking on buttons

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
nope
#### Should this PR be mentioned in the release notes?
yes
#### Is this change backwards compatible or is it a breaking change?
backwards compatible
